toolchain/docs/lower.md

```diff
@@ -23,3 +24,26 @@ can first generate type information for function arguments.
 Lowering is done per `SemIR::InstBlock`. This minimizes changes to the
 `IRBuilder` insertion point, something that is both expensive and potentially
 fragile.
+
+## Mangling
+
+Part of lowering is choosing deterministically unique identifiers for each
+lowered entity to use in platform object files. Any feature of an entity (eg:
+the namespace it appears in, parameters for overloaded functions, etc) that
```

Comment:
```suggestion
lowered entity to use in platform object files. Any feature of an entity (such as
the namespace it appears in, or parameters for overloaded functions) that
```

We're using [Google's developer style guide](https://developers.google.com/style) for documentation. One of the suggestions it makes is to avoid latin abbreviations, [particularly e.g. and i.e.](https://developers.google.com/style/word-list#eg). It also [discourages etc](https://developers.google.com/style/word-list#etc). The "such as ... or" phrasing here follows the first suggestion from the "etc" advice.

The comma is optional, but feels like it may help readability. You might also consider a slightly different phrasing that might not need a comma:

```suggestion
lowered entity to use in platform object files. Any feature of an entity (such as
parent namespaces or overloaded function parameters) that
```

Reply:
Ended up removing most of this and enumerating things more explicitly rather than "eg/such as".

Reply:
Note this sentence is still here.